docs/changelog.rst

```diff
@@ -1,6 +1,7 @@
 CHANGELOG
 =====================================
 
+| February 6, 2021 : feat: Scheduler support `#499 <https://github.com/mergeability/mergeable/pull/499>`_
```

Comment:
Don't necessarily view this as a new feature though. Just that we broke it with the upgrade of probot hence it's more of a fix.